### ISSUE TITLE
Kogan SmarterHome™ Smart Plug (NEW) (V1.2) missing power monitoring

### ISSUE THREAD

**Issue by JinksKid** [tuya/tuya-home-assistant#379]
**Device normal info**
- Kogan
- Kogan SmarterHome™ Smart Plug（NEW)

**Device info (please complete the following information, which can be found in [log](https://github.com/tuya/tuya-home-assistant/wiki/How-to-get-the-log)):**
like this:
{
      "active_time": 1629246986,
      "biz_type": 0,
      "category": "cz",
      "create_time": 1629246986,
      "icon": "smart/icon/ay1550563759866RUJXC/cbd7a90f7e47dd8f5c0589737d1ee211.png",
      "id": "xxxxxxxxxxxxxxxx",
      "ip": "xxxxxxxxxxxxxxxx",
      "lat": "xxxxxxxxxxxxxxxx",
      "local_key": "xxxxxxxxxxxxxxxx",
      "lon": "xxxxxxxxxxxxxxxx",
      "model": "KASPEMHA",
      "name": "Kogan Plug 1",
      "online": true,
      "owner_id": "34885790",
      "product_id": "j4q3nwrus0ts3mde",
      "product_name": "Kogan SmarterHome™ Smart Plug（NEW)",
      "status": [
        {
          "code": "switch_1",
          "value": true
        }
      ],
      "sub": false,
      "time_zone": "+10:00",
      "uid": "xxxxxxxxxxxxxxxx",
      "update_time": 1629315260,
      "uuid": "xxxxxxxxxxxxxxxx"
    },

2021-08-19 08:31:49 DEBUG (SyncWorker_3) [tuya iot] Response: {
  "result": {
    "category": "cz",
    "functions": [
      {
        "code": "switch_1",
        "type": "Boolean",
        "values": "{}"
      }
    ],
    "status": [
      {
        "code": "switch_1",
        "type": "Boolean",
        "values": "{}"
      }
    ]
  },
  "success": true,
  "t": 1629325909115
}

Missing cur_current, cur_power, cur_voltage from these responses.

Thanks

**Comment by tsutsuku** [comment]
You can create a ticket in the Tuya service console with your device specifications.
https://service.console.tuya.com/8/2/create?step=3&id=01030503&source=content_feedback.

**Comment by JinksKid** [comment]
Yes. It was added within a few hours. Tho it still not useful in home assistant as it does not update frequently enough
